cmd/lncli/commands.go

```diff
@@ -1990,6 +1990,12 @@ var listChannelsCommand = cli.Command{
 			Name:  "private_only",
 			Usage: "only list channels which are currently private",
 		},
+		cli.StringFlag{
+			Name: "with",
```

Comment:
`with_peer` may be a better name, wdyt?

Reply:
yeah i considered a more descriptive name but then thought it sounded nice when read like a sentence. i don't have a strong opinion either way, i'll let others chime in with their preference and then go with herd 👍

Reply:
sgtm

Reply:
I would do `--peer`. The "with-" prefix isn't what we usually do.

Reply:
done